### PR TITLE
Repo setup

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

File: app/controllers/FileUploadController.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package controllers
+
+//import com.google.inject.Inject
```
Comment (oliverslade) [inline app/controllers/FileUploadController.scala]: remove unused imports

File: app/config/AppConfig.scala [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+@ImplementedBy(classOf[AppConfigImpl])
+trait AppConfig extends ServicesConfig {
+
+  lazy val callbackUrl = baseUrl("request-corporation-tax-number")
+
+}
+
+@Singleton
+class AppConfigImpl extends AppConfig
```
Comment (oliverslade) [inline app/config/AppConfig.scala]: no new line

File: conf/app.routes [hunk 3]
```diff
@@ -1,3 +1 @@
 # microservice specific routes
```
Comment (oliverslade) [inline conf/app.routes]: it has no routes?? How does the service interact with the stubs?

File: conf/pdfService.routes [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# microservice app routes
+
+POST       /generate        @controllers.PdfServiceController.generate
```
Comment (oliverslade) [inline conf/pdfService.routes]: new line

File: test/controllers/FileUploadControllerSpec.scala [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+  val mockWSClient = mock[WSClient]
+
+  def createSUT = new SUT
+  class SUT extends FileUploadController(mockWSClient, new AppConfigImpl) {
+
+  }
+
+
+}
```
Comment (oliverslade) [inline test/controllers/FileUploadControllerSpec.scala]: new line

File: conf/app.routes (hunk 3 above)
Comment (MatthewAlexanderBeck) [reply conf/app.routes]: This needs removing as routes are in prod.routes

**Pull request merged**